docs/docs/auth/supertokens.md

```diff
@@ -72,3 +113,7 @@ Clicking sign up should navigate you to `/auth` where SuperToken's default login
 <img width="463" height="696" alt="SuperTokens default UI" src="https://user-images.githubusercontent.com/30793/215893664-d367eb3d-566e-4541-a01a-5772d95cc9c7.png" />
 
 After you sign up, you should be redirected back to your Redwood app, and you should see `{"isAuthenticated":true}` on the page.
+
+## Troubleshooting
+
+If going to `http://localhost:8910/auth` results in the plain Javascript file being served instead of the expected auth page, rename the `web/src/auth.tsx` file to `web/src/authentication.tsx`, and update the imports (related to https://github.com/redwoodjs/redwood/issues/9740).
```

Comment:
Seems like a reasonable solution for the time being? Otherwise the `/auth` route serves a plain javascript file.